src/cascadia/TerminalApp/App.cpp

```diff
@@ -713,14 +714,32 @@ namespace winrt::TerminalApp::implementation
     void App::_UpdateTitle(std::shared_ptr<Tab> tab)
     {
         auto newTabTitle = tab->GetFocusedTitle();
+        const auto lastFocusedProfileOpt = tab->GetFocusedProfile();

```

Comment:
Given that you don't appear to be checking the null-ness of this intermediate or anything, why not just write this all on one line instead of two?

(Also, is it OK to not check it for nullness? What's the value when the optional isn't filled?)

```suggestion
        const auto lastFocusedProfile = tab->GetFocusedProfile().value();
```